extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ItagItem.java

```diff
@@ -165,76 +201,321 @@ public MediaFormat getMediaFormat() {
     private int indexEnd;
     private String quality;
     private String codec;
+    private int targetDurationSec = TARGET_DURATION_SEC_UNKNOWN;
+    private long approxDurationMs = APPROX_DURATION_MS_UNKNOWN;
+    private long contentLength = CONTENT_LENGTH_UNKNOWN;
 
     public int getBitrate() {
         return bitrate;
     }
 
-    public void setBitrate(int bitrate) {
+    public void setBitrate(final int bitrate) {
         this.bitrate = bitrate;
     }
 
     public int getWidth() {
         return width;
     }
 
-    public void setWidth(int width) {
+    public void setWidth(final int width) {
         this.width = width;
     }
 
     public int getHeight() {
         return height;
     }
 
-    public void setHeight(int height) {
+    public void setHeight(final int height) {
         this.height = height;
     }
 
+    /**
+     * Get the frame rate per second.
+     *
+     * <p>
+     * It defaults to the standard value associated with this itag and is set to the {@code fps}
+     * value returned in the corresponding itag in the YouTube player response.
+     * </p>
+     *
+     * <p>
+     * Note that this value is only known for video itags, so {@link #FPS_NOT_APPLICABLE_OR_UNKNOWN} is returned for
+     * non video itags.
+     * </p>
+     *
+     * @return the frame rate per second or {@link #FPS_NOT_APPLICABLE_OR_UNKNOWN}
+     */
+    public int getFps() {
+        return fps;
+    }
+
+    /**
+     * Set the frame rate per second.
+     *
+     * <p>
+     * It is only known for video itags, so {@link #FPS_NOT_APPLICABLE_OR_UNKNOWN} is set/used for
+     * non video itags or if the sample rate value is less than or equal to 0.
```

Comment:
```suggestion
     * non video itags, or if the sample rate value is <= 0.
```

Reply:
The docs are for coders. I don't think you need to substitute words for easily understood symbols.

Reply:
Indeed, but using `<` and `>` seems to fail JavaDoc deployments (see https://github.com/TeamNewPipe/NewPipeExtractor/commit/52fa2d939ad9b64644a05f92342269e947e89216 and https://github.com/TeamNewPipe/NewPipeExtractor/runs/6048263942?check_suite_focus=true#step:5:330). I will not change this.